vercel.json

```diff
@@ -14,6 +14,7 @@
     { "source": "/submissions/:match*", "destination": "https://backend.codecrafters.io/submissions/:match*" },
     { "source": "/progress/:match*", "destination": "https://backend.codecrafters.io/progress/:match*" },
     { "source": "/users/:user", "destination": "/prerender/users/[user]" },
+    { "source": "/concepts/:concept", "destination": "/prerender/concepts/[concept]" },
```

Comment:
Let's order these alphabetically except the fallback one - as a rule, wherever order doesn't matter let's try to stick to alphabetical ordering.

Reply:
Yep, I'm trying to stick to this convention too, but some things just keep occasionally slipping through